exercises/concept/date-parser/test/date_parser_test.exs

```diff
@@ -98,48 +64,33 @@ defmodule DateParserTest do
 
   describe "numeric pattern for month matches" do
     @tag task_id: 1
-    test "un-padded 1", do: assert("1" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 2", do: assert("2" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 3", do: assert("3" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 4", do: assert("4" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 5", do: assert("5" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 6", do: assert("6" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 7", do: assert("7" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 8", do: assert("8" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 9", do: assert("9" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 10", do: assert("10" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 11", do: assert("11" =~ Regex.compile!(DateParser.month()))
-    @tag task_id: 1
-    test "un-padded 12", do: assert("11" =~ Regex.compile!(DateParser.month()))
+    test "un-padded months" do
+      assert("1" =~ Regex.compile!(DateParser.month()))
+      assert("2" =~ Regex.compile!(DateParser.month()))
+      assert("3" =~ Regex.compile!(DateParser.month()))
+      assert("4" =~ Regex.compile!(DateParser.month()))
+      assert("5" =~ Regex.compile!(DateParser.month()))
+      assert("6" =~ Regex.compile!(DateParser.month()))
+      assert("7" =~ Regex.compile!(DateParser.month()))
+      assert("8" =~ Regex.compile!(DateParser.month()))
+      assert("9" =~ Regex.compile!(DateParser.month()))
+      assert("10" =~ Regex.compile!(DateParser.month()))
+      assert("11" =~ Regex.compile!(DateParser.month()))
+      assert("11" =~ Regex.compile!(DateParser.month()))
```

Comment:
Error carried over from the previous code 😂 
```suggestion
      assert("12" =~ Regex.compile!(DateParser.month()))
```

Reply:
Oh I missed that.
I was super tempted to write a loop at some point ^^